packages/devtools/devtools/src/panels/echo/ObjectsPanel/ObjectsPanel.tsx

```diff
@@ -28,18 +28,32 @@ const textFilter = (text?: string) => {
   };
 };
 
+const describeContent = (obj: EchoReactiveObject<any>): string => {
+  if ('name' in obj) {
+    return obj.name;
+  }
+  if ('content' in obj && typeof obj.content === 'string') {
+    const maxLength = 30;
+    return obj.content.length < maxLength ? obj.content : `${obj.content.substring(0, maxLength)}...`;
+  }
+  return '';
+};
+
 const { helper, builder } = createColumnBuilder<EchoReactiveObject<any>>();
 const columns: TableColumnDef<EchoReactiveObject<any>, any>[] = [
   helper.accessor('id', builder.string({ header: 'id' })),
   helper.accessor((item) => getType(item)?.objectId, {
     id: 'type',
-    meta: { cell: { classNames: textPadding } },
-    size: 220,
+    ...builder.string(),
+  }),
+  helper.accessor((item) => describeContent(item), {
+    id: 'content',
+    ...builder.string(),
   }),
-  helper.accessor((item) => (item.__deleted ? 'deleted' : ''), {
+  helper.accessor((item) => (isDeleted(item) ? '✅' : '❎'), {
```

Comment:
I might use a red X here to make it easier to distinguish by colour?

Reply:
though about it, but was too much red. I'll probably replace it with smth colorless